### ISSUE TITLE
Fix Issue event emission bug

### ISSUE THREAD

**Issue by kleneway** [kleneway/jacob#12]
### Title
Fix Issue event emission bug

### Description
The `Issue` event is not being emitted properly when a new JACoB coding loop is initiated. The goal is to ensure that the `Issue` event is emitted only once when JACoB starts work for the first time on a brand new issue it hasn't seen before. 

### Requirements
- Ensure the `Issue` event is emitted only once per new issue.
- Update the relevant code to handle the emission correctly.

#### Files to Update:
- `src/server/messaging/queue.ts`

### Acceptance Criteria
- [ ] The `Issue` event is emitted only once per new issue.
- [ ] The code changes are tested and verified to work correctly.

@jacob-ai-bot

Steps to Address Issue: Ensure the 'Issue' event is emitted only once by checking if the event for the specific issue has already been emitted before proceeding to emit again. This might involve adding a check in the event emission logic to verify if the issue has been previously processed.

Files to Update: /src/server/messaging/queue.ts

task assigned to: @jacob-ai-bot
